ddpui/core/ai/smart_chat_processor.py

```
@@ -0,0 +1,746 @@
+
+            response = self.ai_provider.chat_completion(
+                messages=ai_messages, temperature=0.3, max_tokens=1000
+            )
+
+            return EnhancedChatResponse(
+                content=response.content,
+                intent_detected=MessageIntent.DASHBOARD_EXPLANATION,
+                confidence_score=0.8,
+            )
+
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)